File: Lista 6/Python/Questao 5 L6.py
alunos = int(input("Digite o numero de alunos da sua turma: "))
i = 1
media = 0
while i <= alunos :
    print('Digite a nota dos alunos :')
    nota = float(input())
    media = media + nota
    i = i+1

media = media/alunos
print(f"A media da turma é : {media}")

File: Lista 6/C/Questao 5 L6.c
#include <stdio.h>

int main() {
    int alunos;
    int i = 1;
    float media = 0;

    printf("Digite o número de alunos da sua turma: ");
    scanf("%d", &alunos);

    while (i <= alunos) {
        printf("Digite a nota do aluno: ");
        float nota;
        scanf("%f", &nota);
        media = media + nota;
        i = i + 1;
    }

    media = media / alunos;
    printf("A média da turma é: %.2f\n", media);

    return 0;
}
